### DISCUSSION TITLE
New firmware C220

### DISCUSSION THREAD

**Discussion by Nico83500** [JurajNyiri/HomeAssistant-Tapo-Control#738] [Discuss]
Hi,
Today I get a notification from Tapo about a new firmware release for C220 (1.2.2 Build 240914 Rel.55174n). 
I don't know if I can go or not, anyone test it ?
Thank you !

**Comment by JurajNyiri** [comment]
Most of the firmware updates do not affect this integration unless they introduce breaking changes. If you are currently not affected by the cloud password issue, wait until https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/551#issuecomment-2511836647 .

**Reply by Nico83500** [reply]
Thank you, the integration doesn't work with my actual firmware so I'll wait until Friday
